### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wanda-demo**
- add space after instructions to run

### CODE REVIEW

**Comment by wanda-demo** [comment]
FYI @henry-demo

**Comment by wanda-demo** [comment]
need to more a few more changes to readme

**Comment by wanda-demo** [comment]
will create another PR for that

**Comment by wanda-demo** [comment]
```
random message!!
```

**Comment by wanda-demo** [comment]
hey **[@Pullflow](https://app.pullflow.com)** what are some best practices for readme files?

**Comment by henry-demo** [comment]
I'm working on it!

File: README.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ Click this button and it will help you create a new repo, create a new Netlify p
     npm install
     npm run dev
     ```
+
```
Comment (henry-demo) [inline README.md:50]: not needed

**Review by henry-demo — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -47,6 +47,7 @@ Click this button and it will help you create a new repo, create a new Netlify p
     npm install
     npm run dev
     ```
+    
 
   > Alternatively, you can run this locally with [the Netlify CLI](https://docs.netlify.com/cli/get-started/)'s by running the `netlify dev` command for more options like receiving a live preview to share (`netlify dev --live`) and the ability to test [Netlify Functions](https://www.netlify.com/products/functions) and [redirects](https://docs.netlify.com/routing/redirects/).
```
Comment (henry-demo) [inline README.md:52]: we can improve the wording for this

**Review by henry-demo — CHANGES_REQUESTED** [review]
some more changes needed

**Comment by henry-demo** [comment]
**[@Pullflow](https://app.pullflow.com)** can you explain the above syntax?

**Comment by henry-demo** [comment]
@wanda-demo when will you have your other pr ready?

**Comment by henry-demo** [comment]
maybe we can close this one

**Review by henry-demo — COMMENTED** [review]
Can you improve the text for this: Alternatively, you can run this locally with [the Netlify CLI](<https://docs.netlify.com/cli/get-started/)'s> by running the `netlify dev` command for more options like receiving a live preview to share (`netlify dev --live`) and the ability to test [Netlify Functions](<https://www.netlify.com/products/functions>) and [redirects](<https://docs.netlify.com/routing/redirects/>).

**Comment by henry-demo** [comment]
**[@Pullflow](https://app.pullflow.com)** can you share the improved version of the text?

**Comment by henry-demo** [comment]
thank you

**Review by henry-demo — APPROVED** [review]

**Pull request merged**